packages/integrations/src/angular.ts

```diff
@@ -47,6 +47,8 @@ export class Angular implements Integration {
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public constructor(options: { angular?: any } = {}) {
+    logger.warn('You are still using the Angular integration, consider moving to @sentry/angular');
```

Comment:
Not sure if we should place it here without any way to disable it. Some people may not have enough time to move, yet they still would like to use debug without being warned on every refresh :P

Reply:
Changed it to `.log` so only in debug